sap/5.7/modules/ROOT/pages/sap-connector-config-snc-kerberos.adoc

```diff
@@ -0,0 +1,149 @@
+
+* The SAP GUI application. To install this client, Java must already be installed on your machine. Then, download the installer for your operating system from SAP.
+* An SAP ECC instance that can use SNC system variables
+* The following transactions:
+** `STRUST` (Trust Manager)
+** `SM30` (Table Maintenance)
+** `SNC1` (Generate SNC name for user)
+** `SU01` (User Maintenance)
+** `SM59` (RFC Destinations)
+
+== Configure Active Directory
+
+Create a new Service Principal Name (SPN) according to the operating system you are using and export the keytab.
+
+If you are using Windows, follow these steps:
+
+. Create a service user `adminsap` in Active Directory.
+. Check *Password never expires* and uncheck *Use DES encryption types for this account*.
+. Create a Service Principal Name (SPN) `SAPService/sapdev@SSODOMAIN.LAB` for this user in Active Directory.
++
+`setspn -A SAPService/sapdev@SSODOMAIN.LAB DOMAIN_SHORT\sidadm`
++
+. Export the keytab for this SPN from Active Directory. The password is the password to set for the keytab and the SID is the file name of the keytab.
++
+`ktpass -princ SAPService/sapdev@SSODOMAIN.LAB -mapuser DOMAINSHORT\sidadm -crypto AES256-SHA1 -ptype KRB5_NT_PRINCIPAL -mapop set -pass <*password*> -out <SID>aes.keytab`
+
+== Configure the SAP Server
+
+Change the domain and the KDC in the operating system of the SAP Server to set up Kerberos, and add the parameters and the library that authorize you to use SNC:
+
+. Configure the `krb5.conf` file using the KDC address and the realm information (SSODOMAIN.LAB). The `krb5.conf` file is created by default in the `etc` directory. 
++
+----
+[libdefaults]
+    # "dns_canonicalize_hostname" and "rdns" are better set to false for improved security.
+    # If set to true, the canonicalization mechanism performed by Kerberos client may
+    # allow service impersonification, the consequence is similar to conducting TLS certificate
+    # verification without checking host name.
+    # If left unspecified, the two parameters will have default value true, which is less secure.
+    dns_canonicalize_hostname = false
+    rdns = false
+    #   default_realm = EXAMPLE.COM
+        default_realm = DOMAIN.LOCAL
+        
+[realms]
+    #       EXAMPLE.COM = {
+    #                kdc = kerberos.example.com
+    #               admin_server = kerberos.example.com
+    #       }
+DOMAIN.LOCAL = {
+                kdc = DC.DOMAIN.LOCAL
+               admin_server = dc.DOMAIN.local
+       }
+[domain_realm]
+  .domain.local = DOMAIN.LOCAL
+   domain.local = DOMAIN.LOCAL
+
+[logging]
+    kdc = FILE:/var/log/krb5/krb5kdc.log
+    admin_server = FILE:/var/log/krb5/kadmind.log
+    default = SYSLOG:NOTICE:DAEMON
+
+[libdefaults]
+    default_ccache_name = FILE:/tmp/krb5cc_%{uid}
+----
++
+. Import the key to the `etc` directory.
++
+`ktutil`
++
+`wkt /etc/krb5.keytab`
++
+`q`
++
+. Obtain a Kerberos Ticket Granting Ticket (TGT).
```

Comment:
```suggestion
. Obtain a Kerberos Ticket Granting Ticket (TGT):
```

Reply:
(Global - add colon to introduce command lines.)
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-colon(:)